app/views/boxes/_form.html.erb

```diff
@@ -10,6 +10,8 @@
     <%= f.association :shipping_payment %>
     <%= f.input :shipped_at %>
     <%= f.input :shipment_tracking_number %>
+    <%= f.select :assembly_location, Location.pluck(:name, :id),
```

Comment:
Maybe ordering here by location names would turn the user life easier to find the desired record...

Reply:
The sort method is the better way?